lib/repositories/storage_repository.dart

```diff
@@ -14,6 +14,7 @@ class StorageRepository {
   static const String _showFaviconKey = 'show_favicon';
   static const String _showMetadataKey = 'show_metadata';
   static const String _useCustomTabs = 'use_custom_tabs';
+  static const String _useGestures = 'use_gestures';
```

Comment:
I seem to have messed this up for the custom tabs settings as well, but this should have `Key` in its name.

```suggestion
  static const String _useGesturesKey = 'use_gestures';
```

Reply:
Do I change `_useCustomTabs` to `_useCustomTabsKey` as well?

Reply:
Sure, feel free to.